### ISSUE TITLE
Picker with useNativePicker throws warnings on Android

### ISSUE THREAD

**Issue by jvaane** [wix/react-native-ui-lib#1349]
### Describe the bug
When clicking on the picker to open the modal it gives the warning: 
`WARN     WheelPickerMngr(native) at least one component in the wheel picker did not receive the correct font familyNo field mSelectorWheelPaint in class Landroid/widget/NumberPicker; (declaration of 'android.widget.NumberPicker' appears in /system/framework/framework.jar!classes3.dex)` 
&
`WARN     WheelPickerMngr(native) at least one component in the wheel picker did not receive the correct colorNo field mSelectionDivider in class Landroid/widget/NumberPicker; (declaration of 'android.widget.NumberPicker' appears in /system/framework/framework.jar!classes3.dex)`
&
`WARN     WheelPickerMngr(native) at least one component in the wheel picker did not receive the correct font sizeNo field mSelectorWheelPaint in class Landroid/widget/NumberPicker; (declaration of 'android.widget.NumberPicker' appears in /system/framework/framework.jar!classes3.dex)`

### To Reproduce
1. Start android emulator (API level 30)
2. Go to picker component
3. Click to open modal

### Expected behavior
No warnings

### Code snippet
The native example from your examples library produces same warning

### Device (please complete the following information)

 - Device: Android Emulator (Pixel 3)
 - OS: API level 30

### Additional context
only happens to Native Picker. I checked if it was a problem from this lib or from react-native-picker/picker. But I tried their example and there I had no warnings thrown. So that's why I assume it originates here.

**Comment by M-i-k-e-l** [comment]
Hi @jvaane,

I've just tried to run our demo app on a Pixel **_4_** API 30 and I did not get the warning you've mentioned, I doubt the device type matters here.
I'm assuming you are running this on your own app, maybe it's the configuration, can you share your `android/build.gradle` -> `buildscript.ext`?
My example project has:
```
ext {
  buildToolsVersion = "29.0.3"
  minSdkVersion = 21
  compileSdkVersion = 29
  targetSdkVersion = 29
  ndkVersion = "20.1.5948944"
}
```

Also, please try to remove usage of: `labelColor`, `fontFamily` and `fontSize` from the `WheelPicker`, I think this will solve it for now (please do send me the info above so I can reproduce this).

**Comment by M-i-k-e-l** [comment]
I've also tried the following in our demo app (did not reproduce):
```
ext {
   buildToolsVersion = "30.0.3"
   minSdkVersion = 19
   compileSdkVersion = 30
   targetSdkVersion = 30
  RNNKotlinVersion = "1.3.61" // Or any version above 1.3.x
}
```

**Comment by M-i-k-e-l** [comment]
TL;DR - update your `react-native-ui-lib` version.

Details:
We've moved to a new JavaScript WheelPicker from version `5.11.0` ([PR](https://github.com/wix/react-native-ui-lib/pull/1083)).
I did reproduce the issue (it started to happen because of [Restrictions on non-SDK interfaces](https://developer.android.com/guide/app-compatibility/restrictions-non-sdk-interfaces)), but since we've moved to a JavaScript solution, I doubt we'll fix it (since that if you update you'll get the new solution anyway).

@ethanshar, WDYT?
